### PR TITLE
Update changelog and documentation for v0.8.5

### PR DESCRIPTION
**Pull request by idodod**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,25 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.8.5 - 2024-03-11
+
+### Added
+
+- Added `--aws` flag to `RUN` command which makes AWS environment variables or ~/.aws available. [#3803](https://github.com/earthly/earthly/issues/3803)
```
Comment (alexcb) [inline CHANGELOG.md:11]: might be good to include the required `VERSION --flag` here too.

File: docs/earthfile/earthfile.md [hunk 2]
```diff
@@ -1056,6 +1065,15 @@ Specifies the platform to build on.
...
 For more information see the [multi-platform guide](../guides/multi-platform.md).
 
+#### `--allow-privileged` (experimental)
+
+{% hint style='info' %}
+##### Note
+The `--allow-privileged` flag has experimental status. To use this feature, it must be enabled via `VERSION --allow-privileged-from-dockerfile 0.8`.
+{% endhint %}
+
+When the Dockerfile build context points to an earthly artifact reference (e.g. +some-other-build-context/*), the `allow-privileged` flag will allow `RUN` commands under the referenced earthly target to make use of the `RUN --privileged` option.
```
Comment (alexcb) [inline docs/earthfile/earthfile.md:1075]: it's probably better to use the same example target that we use in our hint that appears above.
```suggestion
When the Dockerfile build context points to an earthly artifact reference (e.g. `+mybuildcontext/mydata/*`), the `allow-privileged` flag will allow `RUN` commands under the referenced earthly target to make use of the `RUN --privileged` option.
```

File: docs/earthfile/earthfile.md [hunk 3]
```diff
@@ -1056,6 +1065,15 @@ Specifies the platform to build on.
...
 For more information see the [multi-platform guide](../guides/multi-platform.md).
 
+#### `--allow-privileged` (experimental)
+
+{% hint style='info' %}
+##### Note
+The `--allow-privileged` flag has experimental status. To use this feature, it must be enabled via `VERSION --allow-privileged-from-dockerfile 0.8`.
+{% endhint %}
+
+When the Dockerfile build context points to an earthly artifact reference (e.g. `+mybuildcontext/mydata/*`), the `allow-privileged` flag will allow `RUN` commands under the referenced earthly target to make use of the `RUN --privileged` option.
```
Comment (alexcb) [inline docs/earthfile/earthfile.md:1075]: Can we also add a cautionary note:

> this does not apply to the Dockerfile' RUN --securtiy option (and have a link to https://docs.docker.com/reference/dockerfile/#run---security )

**Review by alexcb — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,10 +4,25 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
...
+### Added
+
+- Added `--aws` flag to `RUN` command which makes AWS environment variables or ~/.aws available. Enable with the `VERSION --run-with-aws` feature flag. [#3803](https://github.com/earthly/earthly/issues/3803)
+- Added `--allow-privileged` flag to `FROM DOCKERFILE` command. Enable with the `VERSION --allow-privileged-from-dockerfile` feature flag. Thanks to [@dustyhorizon](https://github.com/dustyhorizon) for the contribution! [#3706](https://github.com/earthly/earthly/issues/3706)
+
 ### Fixed
 
 - Fixes an issue where wildcard `BUILD`'s are invoked from a relative directory (e.g., an `Earthfile` containing `BUILD ./*+test` invoked with `earthly ./rel-dir+target`). [#3840](https://github.com/earthly/earthly/issues/3840)
 - `--pass-args` will no longer pass builtin args, which would result in `value cannot be specified for built-in build arge errors. [#3775](https://github.com/earthly/earthly/issues/3775)
+- Fixes an issue where `BUILD --auto-skip` is invoked with wildcard builds. [#3862](https://github.com/earthly/earthly/issues/3862)
```
Comment (mikejholly) [inline CHANGELOG.md:18]:
```suggestion
- Fixes a parsing issue with `BUILD` flag arguments and wildcard targets [#3862](https://github.com/earthly/earthly/issues/3862)
```

Comment (idodod) [reply CHANGELOG.md:18]: did you intentionally remove `--auto-skip`?

Comment (mikejholly) [reply CHANGELOG.md:18]: Yes, the issue wasn't related to `--auto-skip` but rather any flag used with wildcards.

Comment (alexcb) [reply CHANGELOG.md]: @mikejholly might be a good idea to quickly update the referenced ticket, in case someone follows this link

Comment (mikejholly) [reply CHANGELOG.md]: Sure. I updated the title. The original report assumed the issue had to do with `--auto-skip`, but the referenced PR describes the actual problem.

**Review by mikejholly — APPROVED** [review]

**Pull request merged**